src/main/java/teammates/ui/controller/InstructorFeedbackQuestionEditAction.java

```diff
@@ -172,67 +165,53 @@ private static FeedbackQuestionAttributes extractFeedbackQuestionData(Map<String
         if (recipientType != null) {
             newQuestion.recipientType = FeedbackParticipantType.valueOf(recipientType);
         }
 
-        String questionNumber =
-                HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                       Const.ParamsNames.FEEDBACK_QUESTION_NUMBER);
+        String questionNumber = getRequestParamValue(Const.ParamsNames.FEEDBACK_QUESTION_NUMBER);
         Assumption.assertNotNull("Null question number", questionNumber);
         newQuestion.questionNumber = Integer.parseInt(questionNumber);
         Assumption.assertTrue("Invalid question number", newQuestion.questionNumber >= 1);
         
         // Can be null
-        String nEntityTypes =
-                HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                       Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIESTYPE);
+        String nEntityTypes = getRequestParamValue(Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIESTYPE);
         
         if (numberOfEntitiesIsUserDefined(newQuestion.recipientType, nEntityTypes)) {
-            String nEntities = HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                               Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIES);
+            String nEntities = getRequestParamValue(Const.ParamsNames.FEEDBACK_QUESTION_NUMBEROFENTITIES);
             Assumption.assertNotNull(nEntities);
             newQuestion.numberOfEntitiesToGiveFeedbackTo = Integer.parseInt(nEntities);
         } else {
             newQuestion.numberOfEntitiesToGiveFeedbackTo = Const.MAX_POSSIBLE_RECIPIENTS;
         }
         
-        newQuestion.showResponsesTo = getParticipantListFromParams(
-                HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                       Const.ParamsNames.FEEDBACK_QUESTION_SHOWRESPONSESTO));
-        newQuestion.showGiverNameTo = getParticipantListFromParams(
-                HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                       Const.ParamsNames.FEEDBACK_QUESTION_SHOWGIVERTO));
-        newQuestion.showRecipientNameTo = getParticipantListFromParams(
-                HttpRequestHelper.getValueFromParamMap(requestParameters,
-                                                       Const.ParamsNames.FEEDBACK_QUESTION_SHOWRECIPIENTTO));
+        newQuestion.showResponsesTo = getParticipantListFromParams(getRequestParamValue(
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWRESPONSESTO));
+        newQuestion.showGiverNameTo = getParticipantListFromParams(getRequestParamValue(
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWGIVERTO));
+        newQuestion.showRecipientNameTo = getParticipantListFromParams(getRequestParamValue(
+                Const.ParamsNames.FEEDBACK_QUESTION_SHOWRECIPIENTTO));
```

Comment:
Sorry, missed this one. :-p

The line break can be at a higher level, just like the other case.